Dockerfile

```diff
@@ -3,6 +3,8 @@ FROM alpine:edge
 RUN apk --update add ruby ruby-dev ruby-bundler && \
     rm -fr /usr/share/ri
 
+RUN apk --update add ca-certificates
```

Comment:
It would be better to add this to line 3, as it will result in one less layer in the image. For a base image like this, you really want to limit layers as much as possible.